Classes/Finisher/MailFinisher.php

```diff
@@ -199,6 +199,7 @@ protected function getRecipientEmailAdresses(string $recipientAddresses): array|
     $recipientAddresses = array_unique(explode(',', $recipientAddresses));
     foreach ($recipientAddresses as $emailOrFieldName) {
       if (GeneralUtility::validEmail($emailOrFieldName)) {
+        $recipientAdded = true;
```

Comment:
on the other hand, wouldn't it be better to get rid of $recipientAdded and just check if $recipientAddressArray is empty at the end?

return !empty($recipientAddressArray) ? $recipientAddressArray : false;